client-app/pages/sign-up/sign-up-page.vue

```diff
@@ -24,10 +29,18 @@
           class="mb-4"
           :label="$t('pages.sign_up.email_label')"
           :placeholder="$t('pages.sign_up.email_placeholder')"
-          type="email"
           is-required
           :error-message="errors.email"
         ></VcInput>
+        <VcInput
+          v-if="registrationKind == 'organization'"
```

Comment:
Maybe it's better to make an enum with these kinds and use it instead of plain strings?